src/main.ts

```diff
@@ -19,6 +19,12 @@ async function main() {
         azPath = await io.which("az", true);
         await executeAzCliCommand("--version");
 
+        let cloud = core.getInput('cloud', { required: false });
+        if (!cloud){

```

Comment:
Instead of saying if(!cloud) and leaving it empty, can we have 
if(cloud) {
// operation
}

Reply:
@eaarora-ms , I made the suggested change.  Please review.